tests/cli/test_integration_datasets.py

```diff
@@ -59,10 +58,6 @@ def test_dataset_import_real_doi(runner, client, doi, sleep_after):
     assert doi['file'] in result.output
     assert doi['creator'] in result.output
 
-    doi_path = urllib.parse.quote(doi['doi'], safe='')
-    metadata_path = client.renku_path / 'datasets' / doi_path / 'metadata.yml'
-    assert metadata_path.exists()
```

Comment:
why remove this?

Reply:
We don't use the DOI anymore so there is no need to check for this specific path (it's a UUID now and we can't check it like this). We also check the proper creation of the dataset in other tests.